release-blocking-jobs.md

```diff
@@ -2,56 +2,127 @@
 
 Here is a list of jobs that should block release if not healthy.
 
-If you want to add a release-branch job, please add them to sig-release-x.x-all dashboard first,
-and show the test is not flaky, then the release team can decide if the test should be added
-to release-blocking dashboard as well.
+If you want to add a release-branch job, please add them to the appropriate
+sig-release-x.y-all dashboard first. Then show the test is not flaky, and the
+release team can decide if the test should be added to release-blocking 
+dashboard as well.
 
-<!-- TODO(krzyzacy): define blocking criteria -->
+## Release-Blocking Criteria
 
-<!-- TODO(krzyzacy): use master-blocking dashboard, once we sort out jobs like scalability -->
+NB: Note that while we describe the "ideal" goals below, not all jobs currently
+in the release-blocking dashboards currently meet these criteria, and we intend
+to correct with follow on PR's.  Please see the umbrella issue for followup
+work: [kubernetes/sig-release#347](https://github.com/kubernetes/sig-release/issues/347)
+
+The ideal release-blocking job meets the following criteria:
+
+- It provisions clusters via open-source software instead of a hosted service
+<!-- TODO(spiffxp): 
+  - should any allowances be made for conformance?
+-->
+- It finishes a run in 120 minutes or less
+<!-- TODO(spiffxp): 
+  - this should be a percentile, not a max 
+-->
+- It runs at least every 3 hours
+- It passes 75% of all of its runs in the past week (regardless of underlying
+  commit(s))
+- Is capable of passing 3 times in a row against the same commit
+- Is owned by a SIG that is response to addressing failures
+<!-- TODO(spiffxp):
+  - represented by a configured testgrid alert
+  - contact info is also part of job description
+-->
+- Fails for no more than 10 runs in a row
```

Comment:
These requirements are a little strange-- suppose something is merged that breaks the tests. What we want to happen is that the release is blocked. What would actually happen, interpreting these rules literally, is that we kick all the tests out.

Reply:
I agree, and would welcome help rewording this.  I'm trying to put the stake in sand before setting it in stone.  The idea is to punt out jobs that are perpetually failing due to a lack of attention.  If they are failing and actively being worked, that's different.  The case could also be made that there should be a standard process to follow of revert-before-fix once an offending commit is identified.

Reply:
I personally think it's actually a bad idea to explicitly spell out the criteria, I think it requires human judgement. I suggest something like:

```
We intend to remove jobs that are getting an insufficient amount of attention from their owners, i.e., they have test problems that aren't being addressed. We won't punt jobs merely because they are failing, we'll punt them when they are not providing good signal (positive OR negative) AND there's no indication that they will begin providing a reliable signal in relatively short (e.g., 1 week) timeframe. There's no shame in being removed, once the problems have been fixed a job can be added back.

In order to prevent the rules from being gamed, we leave it up to humans to determine whether or not a given test meets the criteria.
```

I'd further add that if it's not obvious or is contentious whether a test job meets the criteria, that sig release and sig testing TLs together as a group should make the decision.

I wouldn't bother defining what should happen if that doesn't work, we should just figure it out if it ever happens.

Reply:
We're not going to have bots automatically act on this for quite a while (if ever?).  Think of these more as triggering conditions for human review.  I call out human review as a necessary part of the process to add or remove below. 
 I'd like to get humans out of the business of having to continually scan jobs and metrics, so they have more time to focus on the business of making decisions.  My hope is that by specifying criteria in a manner that can be automated, we can then start focusing more on spelling out the process we should follow.

Reply:
I added some of lavalamp's language and more strongly emphasized that humans have the final say.  I also sketched out a process that we can iterate on in followups